addons/help/gui.hpp

```diff
@@ -42,13 +40,13 @@ class RscDisplayMain: RscStandardDisplay {
         class GVAR(CreditsVersionButton): RscButton {
             onMouseButtonClick = QUOTE(_this call (uiNamespace getVariable 'FUNC(setVersionLine)'));
             idc = IDC_VERSION_BUTTON;
-            colorText[] = {1,1,1,0};
-            colorDisabled[] = {1,1,1,0};
-            colorBackground[] = {0,0,0,0};
-            colorBackgroundDisabled[] = {0,0,0,0};
-            colorBackgroundActive[] = {0,0,0,0};
-            colorFocused[] = {0,0,0,0};
-            soundClick[] = {"",1,1};
+            colorText[] = {1, 1, 1, 0};
+            colorDisabled[] = {1, 1, 1, 0};
+            colorBackground[] = {0, 0, 0, 0};
+            colorBackgroundDisabled[] = {0, 0, 0, 0};
+            colorBackgroundActive[] = {0, 0, 0, 0};
+            colorFocused[] = {0, 0, 0, 0};
+            soundClick[] = {"", 1, 1};
```

Comment:
I like this more without spaces in config. That's what all the debin tools do and how they show up in the config viewer.